tests/unit/test_application.py

```diff
@@ -21,7 +21,7 @@ async def test_expose_with_exposed_endpoints_as_raw_dict(self, mock_conn):
         mock_facade().Expose.return_value.set_result([])
 
         app = Application(entity_id="app-id", model=Model())
-        app.name = "panther"
+        app.model.state.state = {"application": {"app-id": [{"name": "panther"}]}}
```

Comment:
Is this change required because now these attribute aren't settable? Were they before?

Reply:
setting an attribute in user could would have been a mistake, as those were not propagated anywhere. there's no __setattr__.

the test used to set the value directly on the object as a way to monkey-patch the attribute that's needed by the code under test.

arguably it's not a very good test, but doubt that improving tests to represent AllWatcher mechanics better is worthwhile given that we're going to get rid of AllWatcher.